### ISSUE TITLE
[BUG] Option `adopath` was only added to `include` in Stata 14.2+

### ISSUE THREAD

**Issue by mcaceresb** [sergiocorreia/reghdfe#253]
# Bug report

- Stata 13.1 and Stata 14.1
- Arch Linux (kernel 5.16.15-arch1-1) and  Red Hat Enterprise Linux Server (release 7.7)

**Behavior**

- Expected behavior: `reghdfe.ado` should load with no errors
- Actual behavior: `reghdfe.ado` does not load because `adopath` is passed as an option to `include` statements. Example:

```stata
. reghdfe
invalid syntax
(error occurred while loading reghdfe.ado)
r(198);
```

Expected output in this case would be

```stata
. reghdfe
last estimates not found
r(301);
```

which is the case in Stata 15+. The culprit is the `adopath` option, which was introduced in 14.2+.

**Comment by sergiocorreia** [comment]
One question is if we should care or not (I haven't made my mind..)

Google says that 14.2 was released back in 2016, almost 6yrs ago, but more importantly, anyone with Stata 14 should have been able to upgrade freely to 14.2 by now, so  the relevant date of 14.0 was April 2015 (7yrs ago). So I'm guessing you are running something on a server that uses Stata 13?

As a comparison, in R-land [ggplot](https://cran.r-project.org/web/packages/ggplot2/index.html) requires R>=3.3 which was released in 2016.

In terms of alternatives, I can think of three:

1. Fix it. I guess we could replace the adopath option with two lines (one `findfile`, one `include`). Annoying, but not that difficult
2. Ignore it and bump required version to 14.2
3. Tell people to use the old version of reghdfe

Thoughts?

**Comment by mcaceresb** [comment]
@sergiocorreia Yes, sadly I am using Stata in a server that only has 14.1. Assuming these are sensible options to have, if I am being honest I would lean to option 2 and note somewhere what the last reghdfe release that works with 14.1- is.

1 seems annoying for what is admittedly a corner case.
